### DISCUSSION TITLE
rust版本需要使用混淆插件吗？

### DISCUSSION THREAD

**Discussion by om2mo** [shadowsocks/shadowsocks-rust#522] [General]
我的意思是v2ray-plugin好久不更新了，有其他可以用的插件吗？

**Comment by zonyitoo** [comment]
If a method works fine, why would you need to keep switching to others.

**Reply by om2mo** [reply]
It is Bug-fix-only Future development moved to...本来想说这句，发现眼睛走神了

**Reply by zhengkai** [reply]
我还是没太明白需不需要 obfs，我也用 v2ray-plugin，甚至到今天为止 ss-libev + simple-obfs 依然工作正常。我是看 ss-rust 扩展了很多功能（ws、tls），所以想知道正确配置的还需要额外混淆不，毕竟依赖少一个是一个

毕竟这玩意不好测试，裸奔上去真被识别到，一个 ip 就没了……
